plugins/nginx-vhosts/commands

```diff
@@ -49,6 +49,14 @@ case "$1" in
       shopt -u nullglob
     fi
 
+    DOKKU_APP_CIDS=$(get_container_ids $APP)
+    if [[ -n $DOKKU_APP_CIDS ]]; then
+      for DOKKU_APP_CID in $DOKKU_APP_CIDS; do
+        docker cp "$DOKKU_APP_CID:/app/nginx.conf" "$APP_NGINX_TEMPLATE" 2> /dev/null || true
```

Comment:
Attempting to make `$DOKKU_APP_CIDS` an array so that `${DOKKU_APP_CID[0]}` is valid syntax.